code-security/admin_guide/scan-monitor/secrets-scanning.adoc

```diff
@@ -0,0 +1,52 @@
+== Secrets Scanning
+
+You can use Code Security to detect and remove secrets in IaC files stored in your IDEs, Git-based VCS, and CI/CD pipelines.
+
+A secret is a programmatic access key that provides systems with access to information, services or assets. Developers use secrets such as API keys, encryption keys, OAuth tokens, certificates, PEM files, passwords, and passphrases to enable their application to securely communicate with other cloud services. 
+
+For identifying secrets, Prisma Cloud provides default policies that use domain-specific and generic syntax to match on specific signatures and patterns to validate the likelihood or entropy of a string being a secret. You can view the scan aresults directly on *Code Security > Projects* or on the CLI if using Checkov.
+
+The following file types or extensions are scanned for secrets:
+
+* .yml, .yaml
+* .tf
+* .json
+* .bicep
+* build.gradle
+* build.gradle.kts
+* dockerfile
+* gradle.properties
+* go.mod
+* go.sum
+* gemfile
+* pipfile.lock
+* requirements.txt
+* pom.xml
+* package.json
+* package-lock.json
+* yarn
+* yarn.lock
+
+
+[.task]
+=== Suppress Secret Notifications
+
+You have two ways to suppress notifications for a policy violation. You can either https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/prisma-cloud-policies/manage-prisma-cloud-policies[disable] a policy or suppress a notification for a specific resource or repository. As an example, you do not want to be notified of a violation for issues on non-production environments, or for resources without specific tags.
```

Comment:
There are also code suppressions, but I think this is good for what we should put in the docs.